wix/README.md

```diff
@@ -7,6 +7,8 @@
 
 ## How to upgrade to a new OpenJDK version
 
+0. Ensure the necessary dependencies are installed (`wget` and `wix` version 5.0.0 or higher). If you do not have wix installed, you can followe the instructions [here](https://wixtoolset.org/docs/intro/#nettool) to install it.
```

Comment:
I'm now a huge fan of starting a `0`. If you replace them all with `1` markdown will handle the numbering automatically.

Also wget isn't actually a dependency IMO, we just need it in GitHub action to easily pull down some prebuilt binaries.

Reply:
Was that meant to be _not_ a huge fan?

Reply:
> Was that meant to be _not_ a huge fan?

yeah I edited it just before you added this comment :)